CONTRIBUTING.md

```diff
@@ -0,0 +1,269 @@
+<head>
+  <meta charset="UTF-8">
+  <meta name="description" content="Contributing to rocWMMA">
+  <meta name="keywords" content="ROCm, contributing, rocWMMA">
+</head>
+
+# Contributing to rocWMMA #
+
+We welcome contributions to rocWMMA.  Please follow these details to help ensure your contributions will be successfully accepted.
```

Comment:
Rewording: "External contributions are welcome for rocWMMA. Please see the following details to help maximize the likelihood your contributions will be accepted."